src/shared/services/event.service.ts

```diff
@@ -4,20 +4,20 @@ import { environment } from '../../enviroments/enviroment';
 import { Observable, Subscription } from 'rxjs';
 import { Event } from '../interfaces/Event';
 import { getOptions } from '../utils/http-utils';
+import { AuthService } from './auth.service';
 
 const { baseUrl } = environment;
 
 @Injectable({
   providedIn: 'root',
 })
 export class EventService {
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient, private authService: AuthService) {}
 
   // API CALLS------
 
   getEventsForApproval(): Observable<Event[]> {
-    // Login from postman and set your access token from enviroment. We dont have login yet.
-    const options = getOptions();
+    const options = getOptions(this.authService.accessToken);
```

Comment:
Using this const and function in a lot of places in this service. You can put it outside. But double check if the value is changing when the access token changes in the different functions below.